pkg/roles/roles.go

```diff
@@ -147,6 +147,8 @@ var (
 					"clustervolumeclasses",
 					"projectcomputeclasses",
 					"clustercomputeclasses",
+					"quotarequests",
+					"imageallowrules",
```

Comment:
Duplicated `imageallowrules`

Reply:
Ah, must be a leftover merge conflict artifact. Thanks!